tests/trace/data_serialization/test_cases/media_cases.py

```diff
@@ -0,0 +1,255 @@
+            },
+            {
+                "digest": "Aoxws9QUryX0YiZ8ScTAyi4YzX2SO5QHTsLsYABBMjc",
+                "exp_content": VIDEO_BYTES,
+            },
+        ],
+        equality_check=lambda a, b: a.duration
+        == b.duration,  # could do better, but this is a good start
```

Comment:
I think a duration check here is fine. We already test this elsewhere so if it was re-encoding and downscaling quality we'd know, no reason to check frame by frame except to make the test take longer. Can probably remove this comment.